Apps/WebClient/src/ClientApp/src/router.ts

```diff
@@ -241,8 +253,17 @@ router.beforeEach(async (to, from, next) => {
         // Make sure that the route accepts the current state
         const currentUserState = calculateUserState();
         logger.debug(`current state: ${currentUserState}`);
+        const config = store.getters["config/webClient"];
+        logger.debug(`config/webClient: ${config.modules.Dependent}`);
         if (to.meta.validStates.includes(currentUserState)) {
-            next();
+            if (
+                to.meta.requiredModules?.includes(ClientModule.Dependent) &&
+                !config.modules.Dependent
+            ) {
+                next({ path: "/dependentsNotFound" });
```

Comment:
this route does not exist(?)

Reply:
yes, the dependentsNotFound doesn't not exist so that it show the "404 Page not Found".

Reply:
We don't want to route them to a page that does not exist. The handling is designed as a FSM, so I think the default behaviour would suffice

Reply:
updated code to send user to unauthorized page